indra/llimage/llimagebmp.cpp

```diff
@@ -321,11 +321,7 @@ bool LLImageBMP::updateData()
 	if( 0 != mColorPaletteColors )
 	{
 		mColorPalette = new(std::nothrow) U8[color_palette_size];
-		if (!mColorPalette)
-		{
-			LL_ERRS() << "Out of memory in LLImageBMP::updateData()" << LL_ENDL;
-			return false;
-		}
+		llassert_message_return_false(mColorPalette, "Out of memory in LLImageBMP::updateData()")
```

Comment:
Disagree, unless color_palette_size is unreasonable viewer should show a notification (done in a separate commit) and quit then report to bugsplat. If viewer can't allocate this tiny amount of data it's going to crash either way next time it allocates a buffer for something.

Reply:
Should we check whether the requested memory size is really tiny? :-)

Reply:
No, this type of crashes is common when free RAM is actually insufficient. Even if you can squeeze one more image into it, it will fail on the next one.